src/main/java/ch/njol/skript/expressions/ExprParse.java

```diff
@@ -56,7 +56,7 @@
 				"and one that parses the text according to a pattern.",
 		"If the given text could not be parsed, this expression will return nothing and the <a href='#ExprParseError'>parse error</a> will be set if some information is available.",
 		"Some notes about parsing with a pattern:",
-		"- The pattern must be a <a href='../patterns/'>Skript pattern</a>, " +
+		"- The pattern must be a <a href='./patterns/'>Skript pattern</a>, " +
```

Comment:
This page is missing https://en.njol.ch/projects/skript/doc/patterns
Might implement it in this PR or in another PR after tutorials page is ready